### DISCUSSION TITLE
Making relations update/remove opt-in instead of opt-out

### DISCUSSION THREAD

**Discussion by TriPSs** [TriPSs/nestjs-query#128] [Ideas]
It's currently already placed inside the code as a [TODO](https://github.com/TriPSs/nestjs-query/blob/master/packages/query-graphql/src/resolvers/relations/remove-relations.resolver.ts#L19) but wanted to check what others think about this.

Things that I notice in our own API's is that in almost all relations we set `disableUpdate` and `disableRemove`, recently the `update.<options>` and `remove.<options>` (#123) where added to also support relations decorators, descriptions etc.

My main idea is to align the update/remove with the aggregate and make it opt-in, this would change then from this:
```ts
@Relation('todoItem', () => TodoItemDTO, { disableRemove: true })
```

To this:
```ts
@Relation('todoItem', () => TodoItemDTO, { 
   update: { enabled: true }
})
```

**Comment by TriPSs** [comment]
cc: @GP4cK @smolinari

**Reply by GP4cK** [reply]
I'm good with that too.

**Comment by smolinari** [comment]
LGTM. :grin: 

Scott
